qprocessing/receivers.py

```diff
@@ -36,37 +37,53 @@ def set_initconfig_value(sender, **kwargs):
     if len(qpprojects) == 0:
         return None
 
-    toret = {
+    return {
         'qprocessing': {
+            'version': get_version('qprocessing'),
             'gid': f"{kwargs['projectType']}:{kwargs['project']}",
             'async': settings.QPROCESSING_ASYNC_RUN,
             'urls': {
                 'run': f'/qprocessing{__BASE_RUN_MODEL_URL}',
                 'taskinfo': f'/qprocessing{__BASE_TASK_INFO_URL}',
                 'fields': f'/qprocessing{__BASE_ACTION_URL}fields/',
                 'upload': f'/qprocessing{__BASE_UPLOAD_URL}',
+            },
+            'models': [{
+                **QProcessingModel(str(qpp.model.file)).render2dict(),
+                'id': qpp.pk,
+                'results': [],
+                # TODO: move the following into render2dict? (converts [admin] model inputs into [client] form inputs)
+                'inputs': [{
+                    **input,
+                    'visible': True,
+                    'validate': {
+                        'empty': True,
+                        'message': None,
+                        'required': True,
+                        'unique': False,
+                        'valid': False,
+                        '_valid': False,
+                        **input['validate'],
+                    },
+                    'get_default_value': True,
+                } for input in QProcessingModel(str(qpp.model.file)).inputs.values()],
+            } for qpp in qpprojects],
```

Comment:
@wlorenzetti I've added this from client code.

I don't know if the `render2dict` function is only used to populate the client `initConfig.qprocessing.models`.